server/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -30,14 +30,15 @@
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.lucene.uid.Versions;
 import org.elasticsearch.index.VersionType;
+import org.elasticsearch.index.mapper.MapperService;
 import org.elasticsearch.search.fetch.subphase.FetchSourceContext;
 
 import java.io.IOException;
 
 import static org.elasticsearch.action.ValidateActions.addValidationError;
 
 /**
- * A request to get a document (its source) from an index based on its type (optional) and id. Best created using
```

Comment:
I recently merged a PR to deprecate types in `get` requests, so hopefully these changes will already be covered.